modules/sqs/main.tf

```diff
@@ -3,6 +3,15 @@ locals {
   tags = merge(var.tags, { Deployment = var.prefix })
 }
 
+## SQS IAM access policy for metadata-queue.fifo SQS
+## ====================================================================================================
+data "aws_iam_policy_document" "metadata_queue_policy" {
+  statement {
+    actions   = ["sqs:*"] # todo: Lock down access to specific actions and resources.
```

Comment:
Lets make sure there is a card on the backlog to go back and address security for the queues so we can tie the todo to a card on our backlog.

Reply:
Agreed. Added.